src/pages/AboutUs/ManagementCommittee/index.tsx

```diff
@@ -0,0 +1,13 @@
+import styles from './ManagementComittee.module.css'
+import { AspectRatio, Button, ButtonGroup } from '@chakra-ui/react'
+import { Image } from '@chakra-ui/react'
+
+const ManagementComittee = () => {
+  return (
+    <div className={styles.ManagementComittee}>
```

Comment:
Nitpick: I think it would be better to standardize the naming conventions of your css classes. Because in here the first letter is capitalized but in other files, I see that sometimes you don't capitalize the first letter as well. I'm fine with both as long as it is consistent throughout the project :)

<img width="763" alt="Screen Shot 2022-02-07 at 22 29 01" src="https://user-images.githubusercontent.com/69668484/152807311-5642efe4-bb9d-4a3f-9a88-aa5efa070a0b.png">

Reply:
@Ju-Can which one do you prefer?

Reply:
So, have you guys decided which naming convention to use?